Cargo.toml

```diff
@@ -26,6 +26,7 @@ default = []
 cached = "0.55.1"
 chrono = "0.4.40"
 clap = { version = "4.5.32", features = ["derive"] }
+clap-nested-commands = "0.0.6"
```

Comment:
This requires creating a subdirectory with related code for handling each nested command.  At first it annoyed me but then I realized it would let us rework main.rs into a much cleaner file structure

Reply:
oh, i thought we could do it like the exisitng subcommand structure.